torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3761,9 +3787,36 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                                 dtypes=[torch.int],
                                 active_if=IS_WINDOWS),
                    )),
+    UnaryUfuncInfo('conj_physical',
+                   ref=np.conj,
+                   dtypes=all_types_and_complex_and(torch.bool,
+                                                    torch.bfloat16, torch.half),
+                   dtypesIfCPU=None,
+                   dtypesIfCUDA=None,
+                   dtypesIfROCM=None,
+                   skips=(
+                       # File "test_unary_ufuncs.py", line 289, in test_reference_numerics
+                       #  if not torch.can_cast(numpy_to_torch_dtype_dict[expected.dtype.type], dtype):
+                       # KeyError: <class 'numpy.intc'>
```

Comment:
Interesting. I thought we had fixed this issue. cc @kshitij12345

Reply:
Yes it should work :thinking: .

https://github.com/pytorch/pytorch/blob/db32b69591a5098d2529d93f8f6ce859e1868d9a/torch/testing/_internal/common_utils.py#L408-L413